core/src/main/scala/org/apache/spark/deploy/history/ApplicationCache.scala

```diff
@@ -62,21 +61,25 @@ private[history] class ApplicationCache(
 
     /**
      * Removal event notifies the provider to detach the UI.
-     * @param rm removal notification
+     * @param key removal key
+     * @param value removal value
      */
-    override def onRemoval(rm: RemovalNotification[CacheKey, CacheEntry]): Unit = {
+    override def onRemoval(key: CacheKey, value: CacheEntry,
+        cause: RemovalCause): Unit = {
```

Comment:
Maybe mention RemovalCause in the scaladoc?

Reply:
done